server/routes/curfewAddress/requestMoreAddressChecks.ts

```diff
@@ -0,0 +1,75 @@
+import { Request, Response } from 'express'
+import { AddressService, CaseAdminCaseloadService } from '../../services'
+import { convertToTitleCase } from '../../utils/utils'
+import { validateRequest } from '../../middleware/setUpValidationMiddleware'
+import paths from '../paths'
+import { CheckRequestSummary } from '../../@types/assessForEarlyReleaseApiClientTypes'
+
+export default class RequestMoreAddressChecksRoutes {
+  constructor(
+    private readonly addressService: AddressService,
+    private readonly caseAdminCaseloadService: CaseAdminCaseloadService,
+  ) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const { prisonNumber } = req.params
+    const assessmentSummary = await this.caseAdminCaseloadService.getAssessmentSummary(
+      req?.middleware?.clientToken,
+      prisonNumber,
+    )
+    const addressSummary = await this.addressService.getCheckRequestsForAssessment(
+      req?.middleware?.clientToken,
+      prisonNumber,
+    )
+
+    res.render('pages/curfewAddress/requestMoreAddressChecks', {
+      assessmentSummary: {
+        ...assessmentSummary,
+        name: convertToTitleCase(`${assessmentSummary.forename} ${assessmentSummary.surname}`.trim()),
+      },
+      prisonNumber,
```

Comment:
do you need to pass this separate? could you refer to `assessmentSummary.prisonNumber`?